### PR TITLE
Topic overview

### PR DESCRIPTION
**Pull request by tombak98**
This PR includes:
- the first draft of the topic overview page (no tests yet)
- a modified prayer points component that can now be used across pages. (see [topicpage] for an example on how to use, as well as the file itself in the comments).

### CODE REVIEW

File: components/common/PrayerPoints.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { Container, Card } from "react-bootstrap";
+import {FaPrayingHands} from 'react-icons/fa'
+import { IconContext } from "react-icons/lib";
+import { BsDownload } from "react-icons/bs";
+import Button from "react-bootstrap/Button";
+import { useTranslation } from "next-i18next";
+
+interface prayerProps {
```
Comment (wliu080) [inline components/common/PrayerPoints.tsx:9]: Maybe a question more for @jayjhan8? When do we use type vs interface? I think in my changes I generally use type...

File: components/common/PrayerPoints.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    const subtitle:string = t("prayerSummary.subtitle")
+    const view:string = t("prayerSummary.view")
+    const read:string = t("prayerSummary.read")
+
+    return (
+        <Container data-testid={"prayer-points-container"} className={"d-flex px-6" + (showSubtitle ? " justify-content-center" : " justify-content-start")} id="topic-prayer">
+            <Card className="my-4 shadow d-lg-flex flex-lg-row">
+                {/* temporary height for video, before we finalize videos */}
+                {showImg &&
+                    <Card.Img variant="top" src="/bamboo.jpeg" style={{height:'100%', maxWidth:'600px'}}></Card.Img>
```
Comment (wliu080) [inline components/common/PrayerPoints.tsx:31]: Would prefer we start looking at replacing style options with stuff like h-100. Not sure how that would apply with width, but it also has stuff like w-75, etc. We should test and try if we can do w-xl-50 etc

Comment (tombak98) [reply components/common/PrayerPoints.tsx:31]: Oops! yeah that was a bit of rust from not using bootstrap for a bit. I am trying my best to use the bootstrap classes as much as possible and just slipped up here. Please let me know if you catch anything else like this that can be changed to bootstrap.

File: components/common/PrayerPoints.tsx (hunk 1 above)
Comment (tombak98) [reply components/common/PrayerPoints.tsx:9]: Yeah I'm not sure, interface is what we use at my workplace for typing props and that's just what I've been going with, so honestly im not sure but maybe Jay can speak to that?

**Review by wliu080 — APPROVED** [review]

**Pull request merged**